mediator-coordination/src/plugin.rs

```diff
@@ -33,12 +88,46 @@ impl Plugin for MediatorCoordinationPlugin {
 
     fn routes(&self) -> Router {
         let msg = "This should not occur following successful mounting.";
-        let storage_dirpath = std::env::var("STORAGE_DIRPATH").expect(msg);
+        let MediatorCoordinationPluginEnv {
+            public_domain,
+            storage_dirpath,
+            mongo_uri,
+            mongo_dbn,
+        } = load_plugin_env().expect(msg);
```

Comment:
With the new changes merged to main, the mount method now accepts a mutable reference to self, allowing to save this information during mounting, and simply reuse it here. Update to avoid the duplicate call to  `load_plugin_env`.

Reply:
This does not appear to be addressed.